tests/README.md

```diff
@@ -13,6 +13,23 @@ Note: Not all tests have been converted to tox and/or pytest yet.
 
 ## Running the tests
 
+Before running vdsm tests, activate the environment:
+
+    source ~/.venv/vdsm/bin/activate
+
+When done, you can deactivate the environment:
+
+    deactivate
+
+Before running storage tests, make sure you have setup the user storage
+accordingly. To create storage you can do:
+
+    make storage
+
+Read
+[Setting up user storage](./storage/README.md#setting-up-user-storage)
+for more information.
```

Comment:
Read ... for more info is correct, but I think "See ... for more info" would be better.